src/lib/errors/badRequestError.ts

```diff
@@ -0,0 +1,13 @@
+class BadRequestError extends Error {
```

Comment:
could we move all the errors in the same file?

Reply:
Doing so wouldn't impact readability ? As They are all exported from a common index file to avoid multiple import statements. (Just curious to know why)

Reply:
you would still have only one import statement, it would look like `import { ErrorA, ErrorB, ErrorC } from './error'`

Reply:
Yes of course. My question was more about the difference between creating one file per error type and creating one file containing all our errors, or if it was just to have a single file to centralize our errors.

Reply:
I don't know much about performance in node yet but i guess loading such files don't cost too much so i'd go for 1 class per file